selfapi.js

```diff
@@ -120,6 +120,253 @@ API.prototype = {
     }
   },
 
+  // Export all request handler examples as self-test functions.
+  exportAllTests: function (baseUrl, callback) {
+    var self = this;
```

Comment:
Indeed, thanks for the suggestion! But for now, `selfapi` is compatible with `node < v4`, and I'd like to keep it that way.